<<<<<<< HEAD   (4474b0 Revert "lok: text box insertion: default to insert directly )
=======
# error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
--- configure
+++ configure
@@ -24565,6 +24565,7 @@
 cat confdefs.h >>conftest.$ac_ext
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
+#include <stdlib.h>
 typedef long int ac_int64;
 
 /*
@@ -24702,6 +24702,7 @@
 cat confdefs.h >>conftest.$ac_ext
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
+#include <stdlib.h>
 typedef long long int ac_int64;
 
 /*
>>>>>>> CHANGE (010713 Brown paper bag fix for e69f547bce7de376a0af464c5f7af5e7d2c8)
